modules/doc/content/applications/thermal_hydraulics.md

```diff
@@ -0,0 +1,132 @@
+
+These applications are developed as part of the
+[Nuclear Energy Advanced Modeling and Simulation (NEAMS) program](https://neams.inl.gov/).
+
+!media thermal_hydraulics/misc/TH_scales_new.png
+       style=width:50%;display:block;margin-left:auto;margin-right:auto;
+       caption=MOOSE modules support from Reynold-Average Navier Stokes (RANS) Computational Fluid Dynamics (CFD) modeling to 0D lumped parameters modeling.
+       id=TH_Scales
+
+## MOOSE-based Modules for Thermal-Hydraulics Modeling
+
+| Module                                                                  | Scale                                  | Flow-Formulation                                                                                                                                             | Dimension                                                          | Typical Element Count | Typical Runtime | Typical Simulations                                                                                                                                                                                    |
+| :---------------------------------------------------------------------- | :------------------------------------- | :----------------------------------------------------------------------------------------------------------------------------------------------------------- | :----------------------------------------------------------------- | :-------------------- | :-------------- | :----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| [Navier-Stokes Modue](navier_stokes/index.md)                           | Coarse-Mesh CFD \\ \\ RANS simulations | Incompressible, Weakly-Compressible, or Fully-Compressible\\ \\ Single- or Multi-Phase \\ \\ Single- or Multi-Component Flow                                 | Typically, 2D, 2D axisymmetric, or 3D \\ \\ Can also be used in 1D | 10,000                | 1 minute        | Flow through nuclear reactor core or plena \\ \\ 3D multi-phase flow in pipes \\ \\ Natural convection flow in open cavities                                                                           |
+| [Subchannel Module](https://subchannel-dev.hpc.inl.gov/site/index.html) | Subchannel Scale                       | Incompressible or Weakly-Compressible \\ \\ Single-Phase \\ \\ Single- or Multi-Component Flow                                                               | Typically, 3D \\ \\ Can be used in 1D and 2D                       | 100,000               | 10 seconds      | Flow development through nuclear reactor fuel assembly \\ \\ Thermal-hydraulics analysis of nuclear reactor assembly blockage \\ \\ Natural convection cooling in nuclear reactors low-flow assemblies |
```

Comment:
Can be used in 1D and 2D makes no sense for subchannel. I would suggest calling it 2.5D or pseudo-3D or 3D.

Reply:
As we discussed in the INL TH meeting in 03/06/24, we set for including only 3D

Reply:
I had written "1D with lateral coupling" - please change to "3D" if you prefer that

Reply:
Changed to 3D